integrationExamples/gpt/esp_example.html

```diff
@@ -0,0 +1,210 @@
+<html>
+
+    <head>
+        <link rel="icon" type="image/png" href="/favicon.png">
+        <script async src="//www.googletagservices.com/tag/js/gpt.js"></script>
+        <script async src="prebid.js"></script>
+
+        <script>
+            var div_1_sizes = [
+                [300, 250],
+                [300, 600]
+            ];
+            var div_2_sizes = [
+                [728, 90],
+                [970, 250]
+            ];
+            var PREBID_TIMEOUT = 1000;
+            var FAILSAFE_TIMEOUT = 3000;
+
+            var adUnits = [
+                {
+                    code: '/19968336/header-bid-tag-1',
+                    mediaTypes: {
+                        banner: {
+                            sizes: div_2_sizes
+                        }
+                    },
+                    bids: [{
+                        bidder: 'appnexus',
+                        params: {
+                            placementId: 13144370
+                        }
+                    }, {
+                        bidder: 'pubmatic',
+                        params: {
+                            publisherId: '5890',
+                            adSlot: '/43743431/DMDemo@728x90:0',
+                            pmzoneid: 'zone1',
+                            lat: '40.712775',
+                            lon: '-74.005973',
+                            yob: '1982',
+                            kadpageurl: 'www.yahoo.com',
+                            gender: 'M',
+                            kadfloor: '1.75',
+                            bcat: ['IAB1-1', 'IAB1-2']
+                        }
+                    }]
+                }
+            ];
+
+            // ======== DO NOT EDIT BELOW THIS LINE =========== //
+            var googletag = googletag || {};
+            googletag.cmd = googletag.cmd || [];
+            googletag.cmd.push(function() {
+                googletag.pubads().disableInitialLoad();
+            });
+
+            var pbjs = pbjs || {};
+            pbjs.que = pbjs.que || [];
+
+            pbjs.que.push(function() {
+                pbjs.addAdUnits(adUnits);
+                pbjs.setConfig({
+                    userSync: {
+                        filterSettings: { iframe: { bidders: '*', filter: 'include' } },
+                        enabledBidders: ['pubmatic'],
+                        syncDelay: 3000,
+                        userIds: [{
+                            name: 'id5Id',
+                            params: {
+                                partner: 173,            // change to the Partner Number you received from ID5
+                                abTesting: {             // optional
+                                    enabled: true,         // false by default
```

Comment:
I would suggest not enabling this to avoid the id5 id not always being present in the test.